src/components/organisms/SchedulePageModal/SchedulePageModal.tsx

```diff
@@ -1,162 +1,118 @@
-import React, { useState, useMemo, FC } from "react";
-import { startOfDay, addDays, isWithinInterval, endOfDay } from "date-fns";
+import React, { useState, useMemo, FC, MouseEventHandler } from "react";
+import { addDays, startOfToday, format, getUnixTime } from "date-fns";
 import { range } from "lodash";
-
-import { AnyVenue, VenueEvent } from "types/venues";
-
-import { formatDate, formatDateToWeekday } from "utils/time";
-import { WithId, WithVenueId } from "utils/id";
-import { itemsToObjectByIdReducer } from "utils/reducers";
-import { isEventLiveOrFuture } from "utils/event";
+import classNames from "classnames";
 
 import { useConnectRelatedVenues } from "hooks/useConnectRelatedVenues";
 import { useVenueId } from "hooks/useVenueId";
+import { useUser } from "hooks/useUser";
+
+import { Schedule } from "components/molecules/Schedule";
+import { ScheduleVenueDescription } from "components/molecules/ScheduleVenueDescription";
 
-import { EventDisplay } from "components/molecules/EventDisplay/EventDisplay";
+import { ScheduleDay } from "components/molecules/Schedule/Schedule.types";
 
-type DatedEvents = Array<{
-  dateDay: Date;
-  events: Array<WithVenueId<VenueEvent>>;
-}>;
+import {
+  extendRoomsWithDaysEvents,
+  isEventLaterThisDay,
+  prepareForSchedule,
+} from "./utils";
+
+import "./SchedulePageModal.scss";
 
 const DAYS_AHEAD = 7;
```

Comment:
settings

Reply:
Possibly a more descriptive name?

Reply:
moved to settings: https://github.com/sparkletown/sparkle/pull/1302/commits/e5721d0ad131b92d8d01a936e71f0f3a351d9ef1